### DISCUSSION TITLE
Pico +Sim800

### DISCUSSION THREAD

**Discussion by Izqbr** [micropython/micropython#16005] [General]
I used the sim800 driver [https://github.com/pythings/Drivers/blob/master/SIM800L.py](url). I have set up `
            self.uart = UART(0, 9600, timeout=1000, rx=self.MODEM_TX_PIN, tx=self.MODEM_RX_PIN)` `MODEM_TX_PIN  = 0, MODEM_RX_PIN   = 1` and I get an error:  ValueError: bad TX pin. I am transmitting 0 but have to set up Pin(0). 
How to configure it correctly?

**Comment by robert-hh** [comment]
Pin 0 of the RP2 is UART0/tx, to be connected to the modem's rx pin, Pin 1 is UART0/rx, to be connected to the Modem's tx pin. So you have to swap the connections.

**Comment by Izqbr** [comment]
Thank you. I fixed it and now there is no problem!
